nasa/nos3
# [nasa/nos3#378] arducam checkout

Updated arducam to include the fsw/shared folder and a minimally functional standalone or checkout application.

Requires the following submodules to be merged prior:
* https://github.com/nasa-itc/arducam/pull/7

Closes #378 


Testing:
* Build
  * git checkout nos3#378-arducam-checkout
  * git submodule update --init --recursive
  * make
  * make debug
  * cd ./components/arducam/fsw/standalone
  * mkdir build
  * cd build
  * cmake .. -DTGTNAME=cpu1
  * make
  * exit
  * make checkout
* Test
  * In the arducam checkout, try a NOOP
  * Confirm in the Arducam sim that both I2C and SPI appear
 
![image](https://github.com/user-attachments/assets/a14ca2a9-20d3-47c2-9534-2166b9aa7ac3)
